### ISSUE TITLE
Color changes done in a concept table does not update in a published graphic when changed

### ISSUE THREAD

**Issue by tboeni** [visualize-admin/visualization-tool#1203]
When using the links to graphics provided by the publish function of visualize, updates to color schemes are not happening if they are done by a concept table. A case of this is here in this [Cube](https://int.cube-creator.lindas.admin.ch/app/cube-projects/cube-project!!umweltpanelw7q20x3-zayyrl24dv9/csv-mapping). When using the link to an image that was done while an old color scheme was active it will keep this one https://int.visualize.admin.ch/de/v/kzgQcScg9aEo but it really should look like this right now  https://int.visualize.admin.ch/de/v/xnf88WuNFK_L
Changes in data are displayed correctly, but not the update in the concept table. 

- Visualize environment and version: [v3.22.8 ([e15de35](https://github.com/visualize-admin/visualization-tool/commit/e15de359109df353efd3ddd3bc1f56b7963318e1))]
- Browser and version [Edge]

**Comment by bprusinowski** [comment]
Hi @tboeni, thanks for reporting this. The current approach is to fetch the current colors only when a chart is first created – after publishing, we store HEX codes inside the chart config, so that any chart based on a given cube will remain stable in terms of color mapping, even if the colors within the cube are updated.

To avoid confusion, I think this is the right approach, otherwise such changes in the cube could completely change the appearance of many charts. I think it would be possible to implement the "dynamic" color mode, which would always be based on the colors from a cube, but this would be a new feature request / enhancement, as we'd need to think about this design-wise, how to best introduce such a feature and explain it to the user.

@sosiology what do you think?

**Comment by sosiology** [comment]
@bprusinowski I agree that this could lead to confusion for the end user, specifically if the colors of already published charts change. 

@tboeni please let us know if this enhancement has priority for upcoming sprints so we can plan it in accordingly

**Comment by tboeni** [comment]
@sosiology @bprusinowski  Thanks for your answers!
I see your reasoning behind the decision and will have to discuss this befor ethe next sprint with the team :-)
